Cargo.toml

```diff
@@ -33,6 +33,12 @@ boa_macros = { version = "0.16.0", path = "boa_macros" }
 allow_branch = "main"
 
 # The release profile, used for `cargo build --release`.
```

Comment:
I'm guessing this comment is not correct here, right? The release profile is the `profile.release`, and the `profile.ci` should have a different comment.